### PR TITLE
Tweak user input notification to match Figma designs more closely.

### PR DESCRIPTION
**Pull request by asvinb**
## Summary

Addresses issue #2043 

## Relevant technical choices
Update the CSS for the user input notification to match the Figma designs closely.

## Checklist

- [x] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [ ] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [ ] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [ ] My code has proper inline documentation.
- [ ] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

### CODE REVIEW

File: assets/js/components/notifications/UserInputSettings.js [hunk 1]
```diff
@@ -54,6 +54,7 @@ export default function UserInputSettings() {
 			ctaLabel={ __( 'Let’s go', 'google-site-kit' ) }
 			dismiss={ __( 'Remind me later', 'google-site-kit' ) }
 			winImage={ global._googlesitekitLegacyData.admin.assetsRoot + personSitImage }
+			className="googlesitekit-user-input__notification"
```
Comment (aaemnnosttv) [inline assets/js/components/notifications/UserInputSettings.js:57]: Let's move this up next to the `id` as we usually have `className` as one of the first props.

**Review by aaemnnosttv — COMMENTED** [review]
This is looking good @asvinb but it looks like it could use some adjustment for consistency with Figma.

The design shows that the image should still be present at 922px but it disappears at this width in Storybook
image
image

Also note how the text content has a max-width of what seems to be ~50% in this viewport where as it is spanning the whole notification without the image.

I'm noticing as well that the copy used in tablet does not match that in desktop so I'm guessing all viewports weren't updated when that changed, but we should check with @marrrmarrr.

**Comment by asvinb** [comment]
> This is looking good @asvinb but it looks like it could use some adjustment for consistency with Figma.
> 
> The design shows that the image should still be present at 922px but it disappears at this width in Storybook
> image
> image
> 
> Also note how the text content has a max-width of what seems to be ~50% in this viewport where as it is spanning the whole notification without the image.
> 
> I'm noticing as well that the copy used in tablet does not match that in desktop so I'm guessing all viewports weren't updated when that changed, but we should check with @marrrmarrr.

@aaemnnosttv About the 922px, I do think that the designs should align with the existing breakpoints that we have:
https://github.com/google/site-kit-wp/blob/develop/assets/sass/config/_variables.scss#L150-L154
I wouldn't introduce a new breakpoint at 922px only for this particular notification when we have an existing breakpoint at 960px :) Else we'll potentially end up with hundreds of breakpoints :)

**Comment by aaemnnosttv** [comment]
> About the 922px, I do think that the designs should align with the existing breakpoints that we have:

@asvinb Agreed! Although I wasn't suggesting we add or change any breakpoints. The design in Figma shows the image above from under the Tablet section which ends at 960px. With Mobile below it ending at 600px, this is consistent with the current variables you referenced for breakpoints. I was only pointing out how the current component does not show the image at all when the design shows it in tablet viewport. It should only be hidden below tablet as is shown in the design 🙂 

> Also note how the text content has a max-width of what seems to be ~50% in this viewport where as it is spanning the whole notification without the image.

I just wanted to highlight this again as it seems that both tablet and desktop split the content 50/50 with text and image. The image doesn't fill the full half, but occupies the space, the main difference being the "cropped" view of the image in desktop.

File: assets/js/components/notifications/UserInputSettings.js (hunk 1 above)
Comment (asvinb) [reply assets/js/components/notifications/UserInputSettings.js:57]: Updated @aaemnnosttv

**Comment by asvinb** [comment]
> > About the 922px, I do think that the designs should align with the existing breakpoints that we have:
> 
> @asvinb Agreed! Although I wasn't suggesting we add or change any breakpoints. The design in Figma shows the image above from under the Tablet section which ends at 960px. With Mobile below it ending at 600px, this is consistent with the current variables you referenced for breakpoints. I was only pointing out how the current component does not show the image at all when the design shows it in tablet viewport. It should only be hidden below tablet as is shown in the design 🙂
> 
> > Also note how the text content has a max-width of what seems to be ~50% in this viewport where as it is spanning the whole notification without the image.
> 
> I just wanted to highlight this again as it seems that both tablet and desktop split the content 50/50 with text and image. The image doesn't fill the full half, but occupies the space, the main difference being the "cropped" view of the image in desktop.

@aaemnnosttv Image is now showing up for tablets > 600px. Following our conversation we had, I'll open a new ticket about updating the Notification component to be more flexible by adding a new layout prop which can cater for different kind of layouts, such as 50/50 or 75/25, etc

File: assets/js/components/notifications/UserInputSettings.js (hunk 1 above)
Comment (aaemnnosttv) [reply assets/js/components/notifications/UserInputSettings.js:57]: This is still at the bottom here 🙂

**Review by aaemnnosttv — COMMENTED** [review]
This looks good, just the one tiny thing that was missed.

**Comment by aaemnnosttv** [comment]
@googlebot I consent.

**Pull request merged**